src/pattern-matching/let-control-flow.md

```diff
@@ -33,6 +33,7 @@ fn main() {
 }
 ```
 
+# `let else` expressions 
```

Comment:
Matches `# if let expressions` above -- maybe overkill?

Reply:
No, makes sense. I didn't realize that was missing!